.github/CODEOWNERS

```diff
@@ -1 +1 @@
-@BillWagner @jguadagno @khalidabuhakmeh @ovebastiansen @wordshaker @mitchelsellers @CalvinAllen
\ No newline at end of file
+*       @wordshaker @BillWagner @jguadagno @ovebastiansen @CalvinAllen @nicolebuhakmeh @codehippie1
```

Comment:
We'll need to update the team so @nicoleabuhakmeh has write access.

Reply:
@codehippie1 needs adding to the team as well I believe (or at least he was missing from it last night)

Reply:
I can't see how to change @nicoleabuhakmeh's permissions on this repo as I don't have access to the settings. If you can help with the @BillWagner it would be appreciated. She's already a member of the [membership team](https://github.com/orgs/dotnet-foundation/teams/membership/members?page=2)